__tests__/components/AnalysisHeader/AnalysisRenaming.test.tsx

```
@@ -116,4 +143,69 @@ describe("AnalysisRenaming", () => {
+    fireEvent.click(screen.getByTestId("ui5-icon"));
+    const input = screen.getByTestId("ui5-input");
+
+    fireEvent.input(input, { target: { value: "New Analysis Name" } });
+    fireEvent.keyDown(input, { key: "Enter", code: "Enter" });
+
+    expect(mockMutate).not.toHaveBeenCalled();
+  });
+
+  it("should show loading state with placeholder and opacity", () => {
+    const mockUseRenameAnalysis =
+      require("@/hooks/useAnalysis").useRenameAnalysis;
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)